package.json

```
@@ -56,9 +54,10 @@
     "js:build:base": "rollup -c",
     "js:build:min": "cross-env NODE_ENV=production rollup -c",
     "js:build:banner": "foreach -g \"dist/**/*.min.js\" -x \"headr #{path} -o #{path} --version --homepage --author --license\"",
+    "js:build:vue": "vite build && mv dist/bootstrap-table-vue.umd.cjs dist/bootstrap-table-vue.umd.js",
     "js:build": "run-s js:build:*",
-    "css:build:src": "node-sass --output-style expanded src -o src",
-    "css:build:base": "node-sass --output-style expanded src -o dist",
+    "css:build:src": "sass --no-source-map -I node_modules src:src",
+    "css:build:base": "sass --no-source-map -I node_modules -s compressed src:dist",
```

Hi @wenzhixin - thank you for developing `bootstrap-table`.  I have a question about this line: is the compression necessary here?

It seems that `css:build:min` could handle that; if so perhaps a simplification is possible:

```suggestion
    "css:build:base": "sass --no-source-map -I node_modules src:dist",
```

Regards,
James

---

Indeed, there is no need for compression. Thank you for your feedback.

---

You're welcome - thank you.